virttest/qemu_storage.py

```diff
@@ -594,6 +594,7 @@ def _parse_options(self, params):
                 "extent_size_hint",
                 ("qcow2", "raw", "luks"),
             ),
+            "compression_type": (None, "compression_type", ("qcow2",)),
```

Comment:
```suggestion
            "image_compression_type": (None, "compression_type", ("qcow2",)),
```
Suggest adding the prefix `image_` to dedicate this parameter to the image since I am afraid that other components also have the same attribute.

CC @zhencliu